web/src/components/schema/Object.vue

```diff
@@ -1,12 +1,37 @@
 <template>
   <div>
     <template v-for="(val, key) in data">
-      <v-row
-        v-if="val || (!val && !omitEmpty)"
-        :key="key"
-      >
-        <strong class="mr-2">{{ objectKey(key) }}:</strong> {{ renderedValue(val) }}
-      </v-row>
+      <template v-if="isObjectArray(val)">
+        <v-row
+          v-if="val || (!val && !omitEmpty)"
+          :key="key"
+          class="mx-1"
+        >
+          <strong class="mr-2">{{ objectKey(key) }}:</strong>
+        </v-row>
+        <template v-for="item in val">
+          <template v-for="(v, k) in item">
+            <v-row
+              v-if="k !== 'schemaKey'"
+              :key="`${k}`"
+              class="mx-4"
+            >
+              <strong class="mr-2">{{ k }}:</strong> {{ renderedValue(v) }}
+            </v-row>
+          </template>
+          <v-divider :key="`${item}-divider`" />
```

Comment:
I don't think you need this `:key` attribute.

Reply:
if I remove the `:key` it shows linting errors. so I have put it back.